packages/wallets/src/components/WalletsResetMT5Password/WalletSuccessResetMT5Password.tsx

```diff
@@ -28,16 +28,21 @@ const WalletSuccessResetMT5Password: FC<WalletSuccessResetMT5PasswordProps> = ({
 
     const renderButtons = useCallback(() => {
         return (
-            <WalletButton onClick={handleSuccess} size='lg'>
+            <WalletButton isFullWidth={isMobile} onClick={handleSuccess} size='lg'>
```

Comment:
`renderButtons={isMobile ? undefined : renderButtons}` we're putting this condition not to render `renderButtons` if `isMobile` is `true`. 
Is it still required to put `isFullWidth={isMobile}` prop here?